packages/plugin-core/src/utils.ts

```diff
@@ -74,6 +74,15 @@ export class FileUtils {
   static escape(fpath: string) {
     return fpath.replace(/(\s+)/g, "\\$1");
   }
+
+  /**
+   * Given a full fname, return the leaf
+   * @param fname fname
+   * @returns name of leaf node
+   */
+  static getLeafNameFromFname(fname: string) {
```

Comment:
could you add this to [dnode](https://github.com/dendronhq/dendron/blob/master/packages/common-all/src/dnode.ts#L102:L102)? It'll be a good place to consolidate all node path methods